[IMP] inventory: update batch picking screenshots (#5496)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/misc/batch_transfers.rst

=============
Batch picking
=============

> *Batch picking* enables a single picker to handle multiple orders at once, reducing the travel time
> to the same location. Orders are grouped and consolidated into a picking list. After the picking,
> the batch is taken to an output location where the products are sorted into their respective
> delivery packages.

Reviewer: ```suggestion
*Batch picking* enables a single picker to handle multiple orders at once, reducing the time needed
to navigate to the same location in a warehouse.

When picking in batches, orders are grouped and consolidated into a picking list. After the picking,
the batch is taken to an output location where the products are sorted into their respective
delivery packages.
```

.. seealso::
    - :ref:`Using barcode for picking <inventory/management/barcode_picking>`

Because orders must be sorted at the output location after being picked, this picking method suits

> businesses with a few products that are often ordered. Storing high-demand items in easily

Reviewer: ```suggestion
businesses with a few products that are ordered often. Storing high-demand items in easily
```

> accessible locations can increase the number of orders fulfilled efficiently.

Reviewer: ```suggestion
accessible locations can increase the number of orders that are fulfilled efficiently.
```

Next, in the :guilabel:`Transfers` list, click :guilabel:`Add a line` to open the :guilabel:`Add:
Transfers` window. If the :guilabel:`Operation Type` field was filled, the list will filter transfer
records matching the selected :guilabel:`Operation Type`. Click the :guilabel:`New` button to create
a new transfer. Once the transfer records are selected, click :guilabel:`Confirm` to confirm the
batch picking.

.. note::
   For reviewers; Remove this before publishing (:
   Create a SO and confirm it after setting the Outgoing shipment option to 2 or 3 steps to generate
   a Pick entry for the example in runbot. This is because if the runbot was originally configured
   in 1-step, there would be no entries in the newly created operation type, :guilabel:`Pick`!

.. example::

   A new batch transfer assigned to the :guilabel:`Responsible`, `Joel Willis` for the `Pick`
   :guilabel:`Operation Type`. The :guilabel:`Scheduled Date` is set to August 11.

    .. image:: batch_transfers/batch-transfer-form.png
        :align: center
        :alt: View of *Batch Transfers* form.

    Clicking the :guilabel:`Add a line` button opens the :guilabel:`Add:Transfers` window,

>     displaying only picks because :guilabel:`Operation Type` was set to `Pick` on the batch transfer

Reviewer: ```suggestion
    displaying only pickings because the :guilabel:`Operation Type` was set to `Pick` on the batch transfer
```
Either that or "picking operations"

    form. Two pick transfer records are selected by clicking the designated checkboxes. Then, click

>     :guilabel:`Select` to choose these records.

Reviewer: This sentence is a little unclear, can we make it sound more instructional?

>     form. Two pick transfer records are selected by clicking the designated checkboxes. Then, click
>     :guilabel:`Select` to choose these records.

Author: ```suggestion
    form. Click the checkbox to the left of the transfers, `WH/PICK/00001` and `WH/PICK/00002`, to include them in the new transfer. Then, click the
    :guilabel:`Select` button to close the :guilabel:`Add:Transfers` window.
```

    .. image:: batch_transfers/add-transfers-window.png
       :align: center
       :alt: Select multiple transfers from the *Add:Transfers* window.

Add batch from transfers list
-----------------------------

Another method of creating batch transfers is available using the :guilabel:`Add to batch` option in
a list. Navigate to the :menuselection:`Inventory app --> Operations` drop-down menu and select any
of the :guilabel:`Transfers` to open a filtered list of transfers.

.. image:: batch_transfers/transfers-drop-down.png
   :align: center
   :alt: Show all transfer types in a drop-down menu: Receipts, Deliveries, Internal Transfers,
         Manufacturings, Batch Transfers, Dropships.

> On the transfers list, select the checkbox to the left of selected transfers to add in a batch.

Reviewer: ```suggestion
On the transfers list, select the checkbox to the left of the selected transfers to add in a batch.
```

:guilabel:`Done` quantities for each product under the :guilabel:`Detailed Operations` tab. Finally,
select :guilabel:`Validate` to complete the picking.

.. tip::

   Be certain that the batch transfer is complete when the :guilabel:`Validate` button is
   highlighted in purple. If the :guilabel:`Check Availability` button is highlighted instead, that
   means that there are items in the batch that are currently not available in stock.

.. _inventory/batch_transfers/example:

.. example::

   In a batch transfer involving products from pickings, `WH/PICK/00001` and `WH/PICK/00002`, the
   :guilabel:`Detailed Operations` tab shows that the product, `Cabinet with Doors` has been picked
   because the :guilabel:`Done` column matches the value in the :guilabel:`Reserved` column.
   However, `0.00` quantities have been picked for the other product, `Cable Management Box`.

   .. image:: batch_transfers/process-batch-transfer.png
      :align: center
      :alt: Show batch transfer of products from two pickings in the *Detailed Operations* tab.

Only in-stock products are visible in the :guilabel:`Detailed Operations` tab. To view the complete

> product list, switch to the :guilabel:`Operations` tab. From the list, the :guilabel:`Demand` column

Reviewer: ```suggestion
product list, switch to the :guilabel:`Operations` tab. On this list, the :guilabel:`Demand` column
```

> indicates the required quantity for the order. The :guilabel:`Reserved` column show the available

Reviewer: ```suggestion
indicates the required quantity for the order. The :guilabel:`Reserved` column shows the available
```

stock to fulfill the order, while the :guilabel:`Done` column specifies the products that have been

> picked and ready for the next step.

Reviewer: ```suggestion
picked, and are ready for the next step.
```

----------------

On the batch transfer form, if the :guilabel:`Done` quantity of the product is less than the
:guilabel:`Reserved`, a pop-up asking whether to :guilabel:`Create Backorder?` shows up. Clicking
the :guilabel:`Create Backorder` button automatically creates a new batch transfer containing the
remaining products. Click :guilabel:`No Backorder` to finish the picking without creating another
batch picking. Click :guilabel:`Discard` to cancel the validation and go back to the batch transfer
form.

.. image:: batch_transfers/create-backorder.png
   :align: center
   :alt: Show the *Create Backorder* pop-up.

.. _inventory/management/barcode_picking:

Process batch transfer: Barcode app
===================================

Created batch transfers are also listed in the :menuselection:`Barcode` app, accessible by selecting
the :guilabel:`Batch Transfers` button. By default, confirmed batch pickings appear on the
:guilabel:`Batch Transfers` page. Click on the desired batch transfer to open the detailed list of
products for the picking.

.. image:: batch_transfers/barcode-batch-transfers.png
   :align: center
   :alt: Show list of to-do batch transfers in *Barcode* app.

> For the chosen batch transfer, follow the instructions at the top of the page in black background.

Reviewer: ```suggestion
For the chosen batch transfer, follow the instructions at the top of the page in the black background.
```
Right?

- :guilabel:`Responsible`: employee assigned to the picking. Leave this field blank if any worker
  can fulfill this picking.
- :guilabel:`Operation Type`: select the operation type the picking is categorized under from the
  drop-down menu.
- The :guilabel:`Scheduled Date` at the top of the form specifies the date the
  :guilabel:`Responsible` person should complete the transfer to the output location by.

Next, in the :guilabel:`Transfers` list, click :guilabel:`Add a line` to open the :guilabel:`Add:
Transfers` window. If the :guilabel:`Operation Type` field was filled, the list will filter transfer
records matching the selected :guilabel:`Operation Type`. Click the :guilabel:`New` button to create
a new transfer. Once the transfer records are selected, click :guilabel:`Confirm` to confirm the
batch picking.

.. note::
   For reviewers; Remove this before publishing (:
   Create a SO and confirm it after setting the Outgoing shipment option to 2 or 3 steps to generate
   a Pick entry for the example in runbot. This is because if the runbot was originally configured
   in 1-step, there would be no entries in the newly created operation type, :guilabel:`Pick`!

> .. example::

Reviewer 2: ```suggestion

```

Reviewer 2: moved this below, but can start the entire example admonition on line 79, but proper formatting

   A new batch transfer assigned to the :guilabel:`Responsible`, `Joel Willis` for the `Pick`

>    :guilabel:`Operation Type`. The :guilabel:`Scheduled Date` is set to August 11.

Reviewer 2: can move this up to start on line 79. no need to have a blank line between the ..example:: and the example, itself.

Next, navigate to the :guilabel:`Actions ⚙️ (gear)` button and click :guilabel:`Add to batch` from
the resulting drop-down menu.

.. image:: batch_transfers/add-to-batch.png
   :align: center
   :alt: Use *Add to batch* button, from the *Action* button's list.

Doing so opens an :guilabel:`Add to batch` pop-up, where the employee :guilabel:`Responsible` for
the picking can be assigned. Choose from the two radio options to add to :guilabel:`an existing
batch transfer` or create :guilabel:`a new batch transfer`. To begin with a draft, select the
:guilabel:`Draft` checkbox. Conclude the process by clicking :guilabel:`Confirm`.

.. image:: batch_transfers/add-to-batch-window.png
   :align: center
   :alt: Show *Add to batch* window to create a batch transfer.

Process batch transfer
======================

Handle batch transfers in the :menuselection:`Inventory app --> Operations --> Batch Transfers`
page. Select the intended transfer from the list, and on the batch transfer form, input the
:guilabel:`Done` quantities for each product under the :guilabel:`Detailed Operations` tab. Finally,
select :guilabel:`Validate` to complete the picking.

> .. tip::

Reviewer 2: ```suggestion

```

Reviewer 2: moved it below, but can bring the whole tip up to start at line 135 -- but there's no need for the blank line between the tip RST and the tip, itself.

>    Be certain that the batch transfer is complete when the :guilabel:`Validate` button is
>    highlighted in purple. If the :guilabel:`Check Availability` button is highlighted instead, that
>    means that there are items in the batch that are currently not available in stock.

Reviewer 2: ```suggestion
.. tip::  
   Be certain the batch transfer is complete when the :guilabel:`Validate` button is highlighted in
   purple. If the :guilabel:`Check Availability` button is highlighted instead, that means there are
   items in the batch that are currently *not* available in-stock.
```

For the chosen batch transfer, follow the instructions at the top of the page in the black
background. Begin by scanning the product's barcode to record a single product for picking. To
record multiple quantities, click the :guilabel:`✏️ (pencil)` icon and enter the required quantities
for the picking.

.. note::

   Products from the same order are labeled with the same color on the left. Completed pickings are
   highlighted in green.

.. example::

   In a batch transfer for 2 `Cabinet with Doors`, 3 `Acoustic Bloc Screens`, and 4 `Four Person
   Desks`, the `3/3` and `4/4` :guilabel:`Units` indicate that the last two product pickings are
   complete. `1/2` units of the `Cabinet with Doors` has already been picked and after scanning the
   product barcode for the second cabinet, Odoo prompts the user to `Scan a serial number` to record
   the unique serial number for :ref:`product tracking <inventory/serial_numbers/configure>`.

   .. image:: batch_transfers/barcode-products.png
      :align: center
      :alt: Display products to be picked in barcode view.

Once all the products have been picked, click on :guilabel:`Validate` to mark the batch transfer as

> done.

Author: ```suggestion
:guilabel:`Done`.
```

the :guilabel:`Batch Transfers` button.

By default, confirmed batch pickings appear on the :guilabel:`Batch Transfers` page. On that page,
click on the desired batch transfer to open the detailed list of products for the picking.

.. image:: batch_transfers/barcode-batch-transfers.png
   :align: center
   :alt: Show list of to-do batch transfers in *Barcode* app.

For the chosen batch transfer, follow the instructions at the top of the page in the black
background. Begin by scanning the product's barcode to record a single product for picking. To
record multiple quantities, click the :guilabel:`✏️ (pencil)` icon, and enter the required
quantities for the picking.

.. note::
   Products from the same order are labeled with the same color on the left. Completed pickings are
   highlighted in green.

.. example::
   In a batch transfer for 2 `Cabinet with Doors`, 3 `Acoustic Bloc Screens`, and 4 `Four Person
   Desks`, the `3/3` and `4/4` :guilabel:`Units` indicate that the last two product pickings are
   complete.

   `1/2` units of the `Cabinet with Doors` has already been picked, and after scanning the product

>    barcode for the second cabinet, Odoo prompts the user to `Scan a serial number` to record
>    the unique serial number for :ref:`product tracking <inventory/serial_numbers/configure>`.

Reviewer 3: ```suggestion
```suggestion
   barcode for the second cabinet, Odoo prompts the user to `Scan a serial number` to record the
   unique serial number for :ref:`product tracking <inventory/serial_numbers/configure>`.
```
